nightfall-client/src/queues/transform.mjs

```diff
@@ -0,0 +1,31 @@
+import logger from '@polygon-nightfall/common-files/utils/logger.mjs';
+import transform from '../services/transform.mjs';
+import rabbitmq from '../utils/rabbitmq.mjs';
+
+export default function receiveMessage() {
+  rabbitmq.receiveMessage('transform', async message => {
+    const { replyTo, correlationId } = message.properties;
+    const response = {
+      error: null,
+      data: null,
+    };
+
+    try {
+      logger.debug(`Transform queue handler ${message.content.toString()}`);
+
+      const txDataToSign = await transform(JSON.parse(message.content.toString()));
+      logger.debug({
+        msg: 'Returning raw transaction',
+        rawTransaction: JSON.stringify(txDataToSign, null, 2),
```

Comment:
The `JSON.stringify` isn't needed, you can pass the object directly, it is properly handled by the log lib.